gcc/config/riscv/predicates.md

```diff
@@ -212,3 +212,21 @@
 {
   return riscv_gpr_save_operation_p (op);
 })
+
+(define_predicate "register_even_operand"
```

Comment:
I think it is not necessary, and it was removed by commit ecd8198

Reply:
Could you squash your patch together? I mean merge the fixup patch into the original one, since I am review the commit one-by-one, and it would be easier review when you submit patch to upstream.

Reply:
Hi Kito, I have sorted them up.